bundle/libraries/local_path.go

```diff
@@ -77,6 +78,16 @@ func IsLibraryLocal(dep string) bool {
 	return IsLocalPath(dep)
 }
 
+func IsLocalRequirementsFile(dep string) bool {
```

Comment:
This could return the path if it is indeed a local requirements file.

You could also use `dep, ok := strings.CutPrefix` to avoid the `HasPrefix` call.